README.md

```diff
@@ -323,6 +323,16 @@ Returns the first element of an array. Passing n will return the first n element
   // Native
   [1, 2, 3, 4, 5][0];
   // => 1
+  //or
```

Comment:
Maybe add a comment from you here: This is resilient to situations where the input array is not an actual array (like lodash)?

Reply:
I added comments stating that the solution even works for potentially undefined/null/non-array input